guides/installation/overview.md

```diff
@@ -1,14 +1,18 @@
 # Installation overview
 
+## Overview
+
 There are a couple of ways to get Shopware running on your system. Depending on what best suits your development environment, you have the following choices:
 
 * [Docker](./docker.md): The docker installation is the easiest way to get a running Shopware 6.
 * [MAMP](./mamp.md): For quick and easy installation you can also use MAMP tool on mac.
 * [Valet+](./valet.md): You can install Shopware with the epl of Valet+, which is a fork of laravel/valet. 
 * [Dockware](./dockware.md): This is a managed docker setup for Shopware 6 by shopware agency dasistweb.
-* [Installation from scratch](./from-scratch.md): You can install Shopware 6 locally. However, be aware that this will 
+* [Installation from scratch](./from-scratch.md): You can install Shopware 6 locally. However, be aware that this will be the more complex solution since additional or changed system requirements need to be managed by you.
+
+Did you know that there's a video available to this topic? Please take a look:
 
-  be the more complex solution since additional or changed system requirements need to be managed by you.
+{% embed url="https://www.youtube.com/watch?v=nWUBK3fjwVg" %}
```

Comment:
Isn't that the Admin Jest video? I would say that is not part of the installation :)

Reply:
Ah, I've screwed up copying I quess 🙈  I'll search for the correct link and correct it.